src/mito2/src/read/scan_region.rs

```diff
@@ -393,20 +393,29 @@ impl ScanRegion {
             .and_then(|c| c.index_cache())
             .cloned();
 
+        // TODO(zhongzc): currently we only index tag columns, need to support field columns.
```

Comment:
This comment doesn't seem to match the PR title 🤔

Reply:
It's not completely assembled, but this is the key entrance and everything inside is ready.